### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-File] Luminosity Ratio is less than 3:1 for highlighted outline with respective background colour for headers 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1803]
**Actual Result:**
Luminosity Ratio is less than 3:1 for highlighted outline with respective background colour for headers i.e. New bot configuration, Open bot, Open recent, Open transcript, Close tab…etc 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 3:1 for highlighted outline with respective background colour for headers 

**Note:** This issue exits for all headers under file menu 

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to File menu and select it 
3.File menu gets opened 
4.Navigate to the “ New bot configuration, Open bot, Open recent, Open transcript, Close tab…etc 
5. Open 'Colour Contrast Analyser' application.
6. Verify Luminosity Ratio is less than 3:1 for highlighted outline with respective background colour for headers or not
 
**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Tool: Colour contrast Analyzer

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=c3837dea-3bb1-4c5a-9374-19ec384f14c6
![MAS1 4 11_File](https://user-images.githubusercontent.com/53518994/64060363-10bab200-cbe9-11e9-86f8-59978636815d.png)

**Comment by tonyanziano** [comment]
This is another native feature that we have no control over. We would have to write a custom app menu to get around this.

**Comment by Kumar2608** [comment]
Issue got fixed hence we are closing this bug.
